build/patches/fontconfig/0001-Disable-Tests.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/Makefile.am b/Makefile.am
-index cc3024e..e58f868 100644
---- a/Makefile.am
-+++ b/Makefile.am
-@@ -24,7 +24,7 @@
- SUBDIRS=fontconfig fc-case fc-lang src \
- 	fc-cache fc-cat fc-conflist fc-list fc-match \
- 	fc-pattern fc-query fc-scan fc-validate conf.d \
--	its po po-conf test
-+	its po po-conf
```

Comment:
Here I think you just need `install-libLTLIBRARIES` in `src` and `install-pkgconfigDATA` in the root.